### DISCUSSION TITLE
Partial area in line graph

### DISCUSSION THREAD

**Discussion by RobinBertilsson** [recharts/recharts#3813] [Q&A] [answered]
Hi,

What would be the best approach to achieve the following (see image)?
I'm basically trying to create a "partial area" as a part of a line chart whenever the line crosses the reference line (value less than 0).

<img width="910" alt="image" src="https://github.com/recharts/recharts/assets/4040035/7268e4b6-858f-4dee-b15a-1827ef171613">

**Comment by ckifer** [comment, marked as answer]
Hi

Please see

https://release--63da8268a0da9970db6992aa.chromatic.com/?path=/story/examples-areachart--area-chart-fill-by-value
